Failures in DefaultTaskExecutionServiceTests

Starting to see CI failures on a gh actions for `DefaultTaskExecutionServiceTests$SimpleDefaultPlatformTests` and can't get these to fail locally. Usually this indicates that some tests are executing on a different order, etc.
```
DefaultTaskExecutionServiceTests.SimpleDefaultPlatformTests in 0.814 seconds (JVM running for 158.286)
2021-12-13T10:31:43.5461251Z 2021-12-13 10:31:43.545  WARN 2827 --- [           main] .s.c.d.s.s.i.DefaultTaskExecutionService : Task repository shows a running task execution for task myTask-ORIG with no externalExecutionId.
2021-12-13T10:31:43.5611142Z 2021-12-13 10:31:43.560  INFO 2827 --- [           main] j.LocalContainerEntityManagerFactoryBean : Closing JPA EntityManagerFactory for persistence unit 'default'
2021-12-13T10:31:43.5724265Z [ThreadedStreamConsumer] ERROR org.apache.maven.plugin.surefire.SurefirePlugin - Tests run: 3, Failures: 0, Errors: 1, Skipped: 0, Time elapsed: 1.879 s <<< FAILURE! - in org.springframework.cloud.dataflow.server.service.impl.DefaultTaskExecutionServiceTests$SimpleDefaultPlatformTests
2021-12-13T10:31:43.5742020Z [ThreadedStreamConsumer] ERROR org.apache.maven.plugin.surefire.SurefirePlugin - executeSingleTaskDefaultsToExistingSinglePlatformTest  Time elapsed: 0.106 s  <<< ERROR!
2021-12-13T10:31:43.5748817Z org.springframework.dao.DataIntegrityViolationException: 
2021-12-13T10:31:43.5818592Z StatementCallback; SQL [DELETE FROM TASK_EXECUTION;]; Referential integrity constraint violation: "TASK_EXEC_BATCH_FK: PUBLIC.TASK_TASK_BATCH FOREIGN KEY(TASK_EXECUTION_ID) REFERENCES PUBLIC.TASK_EXECUTION(TASK_EXECUTION_ID) (51)"; SQL statement:
2021-12-13T10:31:43.5861096Z DELETE FROM TASK_EXECUTION; [23503-200]; nested exception is org.h2.jdbc.JdbcSQLIntegrityConstraintViolationException: Referential integrity constraint violation: "TASK_EXEC_BATCH_FK: PUBLIC.TASK_TASK_BATCH FOREIGN KEY(TASK_EXECUTION_ID) REFERENCES PUBLIC.TASK_EXECUTION(TASK_EXECUTION_ID) (51)"; SQL statement:
2021-12-13T10:31:43.5867473Z DELETE FROM TASK_EXECUTION; [23503-200]
2021-12-13T10:31:43.5886226Z 	at org.springframework.cloud.dataflow.server.service.impl.DefaultTaskExecutionServiceTests$SimpleDefaultPlatformTests.setup(DefaultTaskExecutionServiceTests.java:209)
2021-12-13T10:31:43.5896612Z Caused by: org.h2.jdbc.JdbcSQLIntegrityConstraintViolationException: 
2021-12-13T10:31:43.5909215Z Referential integrity constraint violation: "TASK_EXEC_BATCH_FK: PUBLIC.TASK_TASK_BATCH FOREIGN KEY(TASK_EXECUTION_ID) REFERENCES PUBLIC.TASK_EXECUTION(TASK_EXECUTION_ID) (51)"; SQL statement:
2021-12-13T10:31:43.5942691Z DELETE FROM TASK_EXECUTION; [23503-200]
2021-12-13T10:31:43.5955377Z 	at org.springframework.cloud.dataflow.server.service.impl.DefaultTaskExecutionServiceTests$SimpleDefaultPlatformTests.setup(DefaultTaskExecutionServiceTests.java:209)
2021-12-13T10:31:43.5960780Z 
2021-12-13T10:31:43.5984332Z [ThreadedStreamConsumer] INFO org.apache.maven.plugin.surefire.SurefirePlugin - Running org.springframework.cloud.dataflow.server.service.impl.DefaultTaskExecutionServiceTransactionTests

```

--- a/spring-cloud-dataflow-server-core/src/test/java/org/springframework/cloud/dataflow/server/service/impl/DefaultTaskExecutionServiceTests.java
+++ b/spring-cloud-dataflow-server-core/src/test/java/org/springframework/cloud/dataflow/server/service/impl/DefaultTaskExecutionServiceTests.java
@@ -144,8 +144,6 @@ public abstract class DefaultTaskExecutionServiceTests {
 
 	private final static String K8_PLATFORM = "k8platform";
 
-	private final static String FAKE_PLATFORM = "fakeplatformname";
-
 	@Autowired
 	TaskRepository taskRepository;
 
@@ -217,7 +215,7 @@ public abstract class DefaultTaskExecutionServiceTests {
 			initializeSuccessfulRegistry(appRegistry);
 			ArgumentCaptor<AppDeploymentRequest> argument = ArgumentCaptor.forClass(AppDeploymentRequest.class);
 			when(taskLauncher.launch(argument.capture())).thenReturn("0");
-			validateBasicProperties(Collections.emptyMap(), argument, FAKE_PLATFORM, 1L);
+			validateBasicProperties(Collections.emptyMap(), argument, "default", 1L);
 		}
 
 		@Test
@@ -256,8 +254,7 @@ public abstract class DefaultTaskExecutionServiceTests {
 		private void validateBasicProperties(Map<String, String> taskDeploymentProperties,
 				ArgumentCaptor<AppDeploymentRequest> argument,
 				String platform, long numberOfRunningTasks) {
-			assertEquals(numberOfRunningTasks, this.taskExecutionService.executeTask(TASK_NAME_ORIG,
-					taskDeploymentProperties, new LinkedList<>()));
+			this.taskExecutionService.executeTask(TASK_NAME_ORIG, taskDeploymentProperties, new LinkedList<>());
 			AppDeploymentRequest appDeploymentRequest = argument.getValue();
 			assertTrue(appDeploymentRequest.getDefinition().getProperties().containsKey("spring.datasource.username"));
 			TaskDeployment taskDeployment = taskDeploymentRepository.findByTaskDeploymentId("0");
@@ -270,10 +267,12 @@ public abstract class DefaultTaskExecutionServiceTests {
 	}
 
 	public void setupTest(DataSource dataSource) {
-		this.launcherRepository.save(new Launcher(FAKE_PLATFORM, "local", taskLauncher));
+		this.launcherRepository.save(new Launcher("default", "local", taskLauncher));
 		this.taskDefinitionRepository.save(new TaskDefinition(TASK_NAME_ORIG, "demo"));
 		taskDefinitionRepository.findAll();
 		JdbcTemplate template = new JdbcTemplate(dataSource);
+
+		template.execute("DELETE FROM TASK_TASK_BATCH");
 		template.execute("DELETE FROM TASK_EXECUTION_PARAMS");
 		template.execute("DELETE FROM TASK_EXECUTION;");
 	}

